library/core/src/mem/mod.rs

```diff
@@ -1106,6 +1106,20 @@ impl<T> fmt::Debug for Discriminant<T> {
     }
 }
 
+#[stable(feature = "discriminant_value", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
```suggestion
#[stable(feature = "discriminant_ord", since = "CURRENT_RUSTC_VERSION")]
```
This needs a new feature name. The same has to be done below.